.github/workflows/python-app.yml

```diff
@@ -31,19 +31,9 @@ jobs:
     - name: Install pip dependencies
       run: |
         python -m pip install --upgrade pip setuptools
-        pip install flake8 isort
-    - name: Lint with flake8
+        pip install -r tests-requirements.txt
```

Comment:
I decided to move the tests requirements into a separate file, so Dependabot can upgrade it manually.